v3/src/data-interactive/handlers/attribute-handler.ts

```diff
@@ -125,7 +126,9 @@ export const diAttributeHandler: DIHandler = {
       if (values?.editable != null) attribute.setEditable(!!values.editable)
       if (values?.formula != null) attribute.setDisplayExpression(values.formula)
       if (values?.name != null) attribute.setName(values.name)
-      if (values?.precision != null) attribute.setPrecision(values.precision)
+      if (hasOwnProperty(values, "precision")) {
+        attribute.setPrecision(values.precision == null || values.precision === "" ? undefined : values.precision)
+      }
```

Comment:
@tealefristoe Note that I only changed this for `precision`, but the `!= null` test is inappropriate for any optional property because it prevents the client from unsetting such optional properties. There should be another PR to address this issue for the other optional properties.